internal/access/organization.go

```diff
@@ -56,14 +56,19 @@ func DeleteOrganization(c *gin.Context, id uid.ID) error {
 	return data.DeleteOrganizations(db, data.ByID(id))
 }
 
-var domainNameReplacer = regexp.MustCompile(`[^\da-zA-Z-]`)
-
 func SanitizedDomain(subDomain, serverBaseDomain string) string {
-	sanitizedDomain := domainNameReplacer.ReplaceAllStringFunc(subDomain, func(s string) string {
-		if s == " " {
-			return "-"
+	replaceSymbols := func(r rune) rune {
+		switch {
+		case r >= 'a' && r <= 'z', r >= '0' && r <= '9':
+			return r
+		case r >= 'A' && r <= 'Z':
+			return unicode.ToLower(r)
+		case r == ' ' || r == '-':
+			return '-'
+		default:
+			return -1 // remove rune from the string
 		}
-		return ""
-	})
-	return strings.ToLower(sanitizedDomain) + "." + serverBaseDomain
+	}
+	slug := strings.Map(replaceSymbols, subDomain)
+	return strings.ToLower(slug) + "." + serverBaseDomain
```

Comment:
Oh, I forgot we did this unconditionally at the end... I guess we don't have to `ToLower` the individual runes